docs/11-decorators.md

```diff
@@ -53,6 +53,48 @@ class PostDecorator
 end
 ```
 
+If you include comments on resources, your decorator class must respond to
+`model` returning a resource, and `decorated?` returning true.
```

Comment:
Are you referring to "comments" as an example association for the example below or ActiveAdmin comments module?

Reply:
I'm referring to  "comments" as an ActiveAdmin comments module.

Reply:
For example, if you have disabled comments as follows,

```ruby
ActiveAdmin.register Post do
  config.comments = false
end
```

your decorator class doesn't require `model` and `decorated?`.

Reply:
Ha, I thought it was the former so glad I ask. Thanks. I'd suggest this to clarify.

```suggestion
If a given resource uses ActiveAdmin's Comments feature, then that resource's decorator class must respond to
`model` where it returns the resource and `decorated?` returns true.
```

Reply:
Thank you very much for your suggestion 🤩